src/index.ts

```diff
@@ -30,7 +30,14 @@ const parseOptions = {
  * https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular#type
  */
 const releaseRulesExtension = [
+  { release: 'patch', type: 'ci' },
   { release: 'patch', scope: 'deps', type: 'chore' },
+  {
+    message: '*typescript*',
+    release: 'patch',
+    scope: 'deps-dev',
+    type: 'chore',
+  },
```

Comment:
Just checking, this ensures that all `chore(deps-dev): *typescript*` messages would trigger a release.

Reply:
Never mind, saw the Pull Request description.

Reply:
correct, `*` is a wild card.

Reply:
So chores don't trigger releases? Why? There might be tons of dev dependencies that affect outputs: faro backend TypeScript and ncc, but look at the front end: literally hundreds of dependencies that change the deliverable.

Reply:
Currently only `dev-deps` dont trigger releases. That's why this rule is added explicitly to allow typescript to release.

Reply:
But why those updates don't trigger releases? They should. It is needed.